eng/Build.props

```diff
@@ -50,8 +50,12 @@
     <!-- Project selection can be overridden on the command line by passing in -projects. -->
     <When Condition="'$(ProjectToBuild)' != ''">
       <ItemGroup>
-        <ProjectToBuild Include="$(ProjectToBuild)" Exclude="@(ProjectToExclude);$(RepoRoot)**\bin\**\*;$(RepoRoot)**\obj\**\*">
-          <RestoreInParallel Condition="'%(Extension)' == '.npmproj'">false</RestoreInParallel>
+        <ProjectToBuild Include="$(ProjectToBuild)"
+            Exclude="@(ProjectToExclude);$(RepoRoot)**\bin\**\*;$(RepoRoot)**\obj\**\*">
+          <BuildInParallel Condition=" '%(Extension)' == '.npmproj' ">false</BuildInParallel>
+          <RestoreInParallel Condition=" '%(Extension)' == '.npmproj' ">false</RestoreInParallel>
+          <!-- Also do not build in parallel w/in npm projects. -->
+          <AdditionalProperties Condition=" '%(Extension)' == '.npmproj' ">BuildInParallel=false</AdditionalProperties>
```

Comment:
Didn't remove all `yarn` contention w/o this line and https://github.com/dotnet/aspnetcore/pull/39945/files#diff-366b25472963c6b33188d84139f6c8997298c5adc857a2624ab0d26a2d87baf0R124.

Some `npm` projects reference other `npm` projects. So, Microsoft.JSInterop.JS.npmproj could contend with signalr.npmproj or signalr-protocol-msgpack.npmproj when they're hit from Microsoft.AspNetCore.Components.Web.JS.npmproj.